README.md

```diff
@@ -1,3035 +1,87 @@
-   * NEW netex_mobilityService_support.xsd
-   * NEW netex_mobilityService_version.xsd
-   * NEW netex_onlineService_support.xsd
-   * NEW netex_onlineService_version.xsd
-   * netex.spp
-
-### 2020.10.04  NewModes:  Revise  __VehicleType__
- * Add __TransportType__ and __PersonalTransportType__.   
- * _Updates to xml schema_:  
-   * NEW netex_fleet_support.xsd
-   * NEW netex_fleet_version.xsd
-   * netex_vehicleType_support.xsd
-   * netex_vehicleType_version.xsd
-   * netex_train_support.xsd
-   * netex_train_version.xsd
-   * netex_vehicleJourney_version.xsd
-   * netex_all_objects_reusableComponents.xsd
-   * NEW netex_all_objects_newModes.xsd
-
-### 2020.10.04  NewModes  Revise Transport Organisations.
- * NewModes: Add  __TransportOrganisation__ and __PublicTransportOrganisation__.   
- * _Updates to xml schema_:   
-   * netex_transportOrganisation_support.xsd
-   * netex_transportOrganisation_version.xsd
-
-### 2020.10.04  NewModes:  Add  references to __ModeOfOperation__.
- * NewModes: Update existing references to __Mode__ to include __ModeOfOperation__.
- * _Updates to xml schema_:   
-   * netex_transportOrganisation_version.xsd
-   * netex_vehicleType_version.xsd
-   * netex_stopPlace_version.xsd
-   * netex_flexiblStopPlace_version.xsd
-   * netex_equipmentTicketing_version.xsd
-   * netex_equipmentSigns_version.xsd
-   * netex_access_version.xsd
-   * netex_line_version.xsd
-   * netex_servicePattern_version.xsd
-   * netex_assistenceBooking_version.xsd
-   * netex_accessRightParameter_version.xsd
-   * netex_usageParameterTravel_version.xsd
-   * netex_fareFrame_version.xsd
-
-### 2020.10.04  NewModes : Revise modes
- * NewModes Add __ModeOfOperation__.  
- * FIX: Also correct typos in __Notice__  file.
- * _Updates to xml schema_:    
-   * NEW netex_modeOfOperation_support.xsd
-   * NEW netex_modeOfOperation_version.xsd
-   * netex_mode_support.xsd
-   * netex_mode_version.xsd
-   * netex_notice_version.xsd
-   * netex.spp
-----
-## Version 1.1.2 - Base version plus further minor fixes comprising.
-
-### 2020.08.11  Update Oxygen project to include new examples
- * _Other updates_:    
-   * netex.xpr    
- * _Updates to examples_:
-   * \examples\standards\era_uic\Netex_Eurostar mapping_era_1.xml
-   * \examples\standards\era_uic\Netex_Eurostar mapping_era_2.xml
-   * \examples\standards\era_uic\Netex_era_uic_joiningsplitting.xml
-   * \examples\standards\era_uic\Netex_era_uic_timetable_hack_01.xml
-   * \examples\standards\norway\stops\BasicStopPlace_example.xml
-
-### 2020.08.11   FIX  Issue #110 Add missing fuel types to  __VehicleType__  /  __FuelType__
- * Additional values:  _electricContact, battery, dieselBatteryHybrid, petrolBatteryHybrid, biodiesel, hydrogen, liquidGas, methane, ethanol_.
-  * _Updates to xml schema_:    
-    * netex_framework\netex_reusableComponents\netex_vehicleType_support.xsd
+- You can refer to the NeTEx UML Physical and Conceptual models for a detailed UML view of the schema packages.
```

Comment:
```suggestion
- You can refer to the NeTEx UML Physical and Conceptual models (note that NeTEx Conceptual model is Transmodel) for a detailed UML view of the schema packages.
```

Reply:
@EliotTerrier  can you accept this one and adapt the other?